test/files/playlist.xml

```diff
@@ -5,20 +5,20 @@
             The full first episode (12 minutes)! On a desolate island, suicidal sheep Franck meets his fate in a quirky salesman, who offers him the gift of a lifetime. Little does he know that he can only handle so much lifetime... "Cosmos Laundromat" started in 2014 as an experimental feature film, in which an adventurous and absurdist love story is being told by multiple teams - each working in their own unique style. The opening of the film, the 10 minutes pilot "First Cycle", has been made in the Netherlands by the studio of Blender Institute in Amsterdam. The film itself and all of the artwork files were made with free/open source software and are available under a permissive license. Free to share, free to remix and free to learn from.
         </description>
         <link>
-            http://content.jwplatform.com/previews/TQjoCPTk-V7MoJd7l
+            http://content.jwplatform.com/previews/1CshXyp0-V7MoJd7l
```

Comment:
@paulmandal looks like you pulled a mediaid from another account. Please update the xml with a more recent source from our shared one like https://cdn.jwplayer.com/v2/playlists/bEhVQYdb?format=mrss

Reply:
Also test/files/playlist.xml is for unit tests, not for actual playback tests.